src/utils/convert.util.ts

```diff
@@ -12,23 +12,31 @@ export function toHex(value: string): Color["hex"] {
     ctx.fillStyle = value;
 
     return ctx.fillStyle;
-  } else if (value.length === 7) {
+  } else if (value.length === 4 || value.length === 5) {
+    let newValue = "#";
+
+    for (let i = 1; i < value.length; i++) {
+      newValue += value[i] + value[i];
+    }
+
+    return newValue;
```

Comment:
What do you think about this change? It does not create variables internally and is more concise in writing.
```ts
value = value
  .split("")
  .map((v, i) => (i === 0 ? "#" : v + v))
  .join("");

return value;
```